src/components/App.tsx

```diff
@@ -26,10 +26,17 @@ import Control from './Control';
 import SplashScreen from './SplashScreen';
 import Store from '../stores/Store';
 import { MessageSendingResponse } from '../models/Message';
+// eslint-disable-next-line import/no-named-as-default
```

Comment:
instead of disabling rule let fix it
you have named export Messages in './MessageList' and you are importing as default Messages
delete named export and error should go away